keras_retinanet/bin/debug.py

```diff
@@ -177,7 +184,12 @@ def run(generator, args):
             image, image_scale = generator.resize_image(image)
             annotations[:, :4] *= image_scale
 
-        anchors = anchors_for_shape(image.shape)
+        # optionally load anchor parameters
+        anchor_params = None
+        if args.config and 'anchor_parameters' in args.config:
+            anchor_params = parse_anchor_parameters(args.config)
```

Comment:
Would it make sense to move this block (line 187-190) outside of the run function into the main (to line 229)? anchor_params would also have to be passed to the run function, as in ` run(generator, args, anchor_params)`, but anchor_params shouldn't change from image to image and therefore doesn't need to be in the loop.